DEV-INTRO.md

```diff
@@ -8,6 +8,8 @@ The recommended workflow is gitflow which is described in the PEcAn documentatio
 
 ## Developing in Docker
 
+If running on a linux system it is recommended to add your user to the docker group. This will prevent you from having to use `sudo` to star the docker containers, and makes sure that any file that is written to a mounted volume is owned by you. This can be done using `sudo adduser ${USER} docker`.
```

Comment:
```suggestion
If running on a linux system it is recommended to add your user to the docker group. This will prevent you from having to use `sudo` to start the docker containers, and makes sure that any file that is written to a mounted volume is owned by you. This can be done using `sudo adduser ${USER} docker`.
```